admin/acceptance_tests/layers_wmts_test.py

```diff
@@ -35,10 +35,10 @@ def insert_layer_wmts_test_data(dbsession):
 class TestLayerWMTS():
 
     def test_view_index_rendering_in_app(self, test_app):
-        expected = [('_id_', ''),
-                    ('name', 'Name'),
-                    ('public', 'Public'),
-                    ('layer', 'WMTS layer name')]
+        expected = [('_id_', '', 'false'),
+                    ('name', 'Name', 'true'),
+                    ('public', 'Public', 'true'),
+                    ('layer', 'WMTS layer name', 'true')]
```

Comment:
The relations wit interfaces, restrictionareas are on layer table, so they may appear in LayerWMTS also.
This may be factorized using an abstract LayerViews class as we will have the same in LayerV1